librz/include/rz_cons.h

```diff
@@ -839,8 +844,8 @@ RZ_API void rz_cons_break_end(void);
 RZ_API void rz_cons_break_timeout(int timeout);
 
 /* pipe */
-RZ_API int rz_cons_pipe_open(const char *file, int fdn, int append);
-RZ_API void rz_cons_pipe_close(int fd);
+RZ_API int rz_cons_pipe_open(const char *file, int fdn, int append, RzConsPipeStack *fds);
+RZ_API void rz_cons_pipe_close(int fd, RzConsPipeStack *fds);
```

Comment:
does these functions allows NULL as value for RzConsPipeStack ? if yes add RZ_NULLABLE or RZ_NOTNULL in the opposite case.